lib/batch_sanitizer.js

```diff
@@ -0,0 +1,41 @@
+var fs = require("fs");
+
+var MINIFIED_AVG_LINE_LENGTH_CUTOFF = 100;
+
+function BatchSanitizer(files, stderr) {
+  this.files = files;
+  this.stderr = stderr || process.stderr;
+}
+
+BatchSanitizer.prototype.sanitizedFiles = function() {
+  var sanitizedFiles = [];
+
+  for(var i = 0; i < this.files.length; i++) {
+    if (this.isMinified(this.files[i])) {
+      this.stderr.write("WARN: Skipping " + this.files[i] + ": it appears to be minified\n");
```

Comment:
That's a valid concern. In manual testing I didn't find false positives, but the experience would be not getting a file analyzed:

- file reported as `skipped` in stderr output for engine on builds page
- no other eslint issues reported for that file on .com
- file still rated